pkgs/applications/editors/vscode/with-extensions.nix

```diff
@@ -52,18 +52,27 @@
 
 let
   inherit (vscode) executableName longName;
+  # This is expensive but I don't see a better way of doing this
+  vscodeExtensions' = (
+    lib.converge (
+      extensions:
+      lib.sort (a: b: a.outPath < b.outPath) (
+        lib.unique (extensions ++ lib.concatMap (x: x.propagatedExtensions or [ ]) extensions)
+      )
+    ) vscodeExtensions
+  );
```

Comment:
Can you please document why we need this? The comment doesn't help.

Reply:
Sure, that seems reasonable.